src/main/java/frc/trigon/robot/poseestimation/apriltagcamera/io/AprilTagLimelightIO.java

```diff
@@ -39,8 +40,9 @@ private void updateHasResultInputs(AprilTagCameraInputsAutoLogged inputs, Limeli
     }
 
     private void updateNoResultInputs(AprilTagCameraInputsAutoLogged inputs) {
-        inputs.visibleTagIDs = new int[0];
         inputs.cameraSolvePNPPose = new Pose3d();
+        inputs.visibleTagIDs = new int[0];
+        inputs.distanceFromBestTag = Double.POSITIVE_INFINITY;
```

Comment:
when you changed this here, you also changed the depending logic in the AprilTagCamera class?